ffi/SDL2_0_h.lua

```diff
@@ -726,6 +726,7 @@ int SDL_SetSurfacePalette(SDL_Surface *, SDL_Palette *) __attribute__((visibilit
 int SDL_LockSurface(SDL_Surface *) __attribute__((visibility("default")));
 void SDL_UnlockSurface(SDL_Surface *) __attribute__((visibility("default")));
 void SDL_SetWindowIcon(SDL_Window *, SDL_Surface *) __attribute__((visibility("default")));
+char * SDL_GetBasePath(void);
```

Comment:
Please also add it to 

https://github.com/koreader/koreader-base/blob/5956ba1cc56a61e0eacbf19e1c54bcbbbf6cc980/ffi-cdecl/SDL2_0_decl.c#L132-L137

I suspect it'd look more like this if you used ffi-cdecl btw:
```suggestion
char *SDL_GetBasePath(void) __attribute__((visibility("default")));
```
@NiLuJe Do we care about that or do we regard it as good enough until someone runs that again, provided it's added to the ffi-cdecl files?

Reply:
The attributes are ignored by LuaJIT (IIRC. I do remember that they're definitely ignored by Python's cFII, because it's a parsing error there ;p) so the last nit is not that big a deal.

But, yeah, I'd definitely want the cdecl stuff ;).

Reply:
Ok. I will add the cdecl stuff but I remember the ffi-cdecl thingy to give me headaches when I had to use it, so probably I will skip the checks and adds things manually.

Reply:
Since I'm going to do that I can add #924 too.